src/main/resources/services-context.xml

```diff
@@ -32,13 +32,9 @@
 		<constructor-arg value="${orchestration.password}"/>
 	</bean>
 
-	<bean class="org.ihtsdo.snowowl.authoring.single.api.service.UiStateService"/>
```

Comment:
It's a bit embarrassing that we still have Spring XML hanging around. Thanks for reducing it.